docs/usage.md

```diff
@@ -29,6 +29,18 @@ system:
 sudo curl -L --output /usr/local/bin/pie https://github.com/php/pie/releases/latest/download/pie.phar && sudo chmod +x /usr/local/bin/pie
 ```
 
+### Docker installation
+
+PIE is published as binary-only Docker image, so you can install it easily during your Docker build:
+
+```Dockerfile
+COPY --from=ghcr.io/php/pie:latest /pie /usr/bin/pie
```

Comment:
I wonder if the image naming should be something like `ghcr.io/php/pie.phar:<tag>`, to help indicate that you can't simply run it (e.g. `docker run --rm ghrc.io/php/pie --version`) :thinking:

Reply:
That's fine IMO, thank you